stf/launch/Remote Debugging - Saros_STF_Alice.launch

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<launchConfiguration type="org.eclipse.pde.ui.RuntimeWorkbench">
+    <booleanAttribute key="append.args" value="true"/>
+    <booleanAttribute key="askclear" value="true"/>
+    <booleanAttribute key="automaticAdd" value="true"/>
+    <booleanAttribute key="automaticValidate" value="false"/>
+    <stringAttribute key="bootstrap" value=""/>
+    <stringAttribute key="checked" value="[NONE]"/>
+    <booleanAttribute key="clearConfig" value="false"/>
+    <booleanAttribute key="clearws" value="false"/>
+    <booleanAttribute key="clearwslog" value="false"/>
+    <stringAttribute key="configLocation" value="${workspace_loc}/.metadata/.plugins/org.eclipse.pde.core/Remote Debugging - Saros_STF_Alice"/>
+    <booleanAttribute key="default" value="true"/>
+    <booleanAttribute key="includeOptional" value="true"/>
+    <stringAttribute key="location" value="${workspace_loc}/../workspace-alice-stf"/>
+    <stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+    <stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="-os ${target.os} -ws ${target.ws} -arch ${target.arch} -nl en_US"/>
+    <stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.pde.ui.workbenchClasspathProvider"/>
+    <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Xmx512m -ea -XX:MaxPermSize=256m -Djava.security.manager &quot;-Djava.security.policy=file:${workspace_loc:saros.stf}/test/resources/all.policy&quot;  -Dsaros.debug=true -Dsaros.testmode=12345 -Dsaros.sleepTime=200 -Djava.rmi.server.hostname=localhost -Dorg.eclipse.swtbot.keyboard.strategy=org.eclipse.swtbot.swt.finder.keyboard.MockKeyboardStrategy  -Dorg.eclipse.swtbot.keyboard.layout=saros.stf.server.bot.default -Duser.language=en -Duser.country=US -Dfile.encoding=UTF-8 -Djava.util.Arrays.useLegacyMergeSort=true&#10;-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005"/>
```

Comment:
You cannot set the size of the PermGeneration since Java 8.

Reply:
Just copied from the existing profile. But I can remove it.